### ISSUE TITLE
Is it related to Mioco?

### ISSUE THREAD

**Issue by vi** [Xudong-Huang/may#12]
The project looks similar to [mioco](https://github.com/dpc/mioco). Is it related somehow?

In any case it should be mentioned somewhere in README, maybe included in the performance comparison.

**Comment by Xudong-Huang** [comment]
Yes, it's related. But I don't think it support all the APIs supplied by `may` and the thread context compatibility. I don't want to raise argues that `May` is better or worse than other implementations. Users can choose on their own.

**Comment by jedisct1** [comment]
It's also similar to Coio and possibly others.

What all these projects show is that a coroutine-based approach is something Rust users really want and enjoy.

**Comment by vi** [comment]
Unless there are too many of them, I expect each of them (except of abandoned ones) to have "See also" section pointing to other competing implementations and mini-comparisons.
